app/manage/index.js

```diff
@@ -60,6 +61,7 @@ function manageRouter (nextApp) {
    * List, Create, Read, Update, Delete operations on teams.
    */
   router.get('/api/teams', listTeams)
+  router.get('/api/my/teams', authenticate, listMyTeams)
```

Comment:
Instead of using the authenticate middleware directly, could you use the "check" middleware and the `can('public')` capability? Maybe we should rename that capability because it means "anyone that is authenticated can use this route"